ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-ssh.j2

```diff
@@ -1,5 +1,50 @@
 {# Management SSH #}
 {% if management_ssh is defined and management_ssh is not none %}
+
+{%  if management_ssh.access_groups is arista.avd.defined %}
+### IPv4 ACL
+
+| IPv4 ACL | VRF |
+| -------- | --- |
+{%      for acl in management_ssh.access_groups %}
+| {{ acl.name }} | {{ acl.vrf | arista.avd.default('-') }} |
+{%      endfor %}
+{%  endif %}
+
+{%  if management_ssh.ipv6_access_groups is arista.avd.defined %}
```

Comment:
If IPv6 section is not displayed, an additional line is added to the final markdown.

To avoid that, it would be nice to invert line 13 and 14:

```j2

{%  if management_ssh.ipv6_access_groups is arista.avd.defined %}
<...>
```

Should be 

```j2
{%  if management_ssh.ipv6_access_groups is arista.avd.defined %}

<...>
```

Should be applied to any if block